### ISSUE TITLE
optimize: seata client support config read from file source in spring boot

### ISSUE THREAD

**Issue by laywin** [apache/incubator-seata#6260]
## Why you need it?
 Is your feature request related to a problem? Please describe in details

~~currently, in spring boot, config read chain is: system env -> spring environment -> default value -> original source read read(file,zk,nacos...depend on config type).~~

currently, when config type is file mode in spring boot, config read chain is: system env -> spring environment -> default value -> original source

if there is a value for the property, search returned. the situation may occur: the property not configed in spring and exist a default for it, so it will result a default value instead of read right property in file configuration.

## How it could be?
A clear and concise description of what you want to happen. You can explain more about input of the feature, and output of it.

**Comment by laywin** [comment]
The temporary idea is：move [SeataPropertiesLoader](https://github.com/apache/incubator-seata/blob/2.x/server/src/main/java/io/seata/server/spring/listener/SeataPropertiesLoader.java) in server to seata-spring-autoconfigure-core and do some 
small adjustments

**Comment by funky-eyes** [comment]
> The temporary idea is：move [SeataPropertiesLoader](https://github.com/apache/incubator-seata/blob/2.x/server/src/main/java/io/seata/server/spring/listener/SeataPropertiesLoader.java) in server to seata-spring-autoconfigure-core and do some small adjustments

I agree with this approach, but the configuration center piece of the subsequent need to refactoring, I do not know whether there is a conflict with the refactoring, @wangliang181230 What do you think?
